ui/pages/confirmations/confirmation/confirmation.js

```diff
@@ -366,7 +367,8 @@ export default function ConfirmationPage({
       try {
         if (useSafeChainsListValidation) {
           const response = await fetchWithCache({
-            url: 'https://chainid.network/chains.json',
+            url: CHAIN_SPEC_URL,
```

Comment:
Looks great! ❤️ 

Was checking the docs for the [package](https://github.com/poowf/eth-chainlist), seems like the package will update itself automatically once a day, does this mean that we can swap the calls to `fetchWithCache` and use `rawChainData` directly?

Reply:
Thank you :)

I guess the idea here is that we have 2 sources:
1. a bundled list, fetched once at build-time via the `eth-chainlist` package
2. the list fetched dynamically from `chainid.network` by `fetchWithCache`

So here those are complimentary. But I guess you're suggesting we might want to consolidate the sources into one single-source-of-truth. If so I agree and think that can very much be worth exploring!

Reply:
Yes, i guess not necessarily part of this PR, thnx!